Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -139,6 +139,16 @@ public RotationConstraintType ConstraintOnRotation
             set => constraintOnRotation = value;
         }
 
+        [SerializeField]
+        [Tooltip("Use world or local space for the constraining on an axis.")]
```

Comment:
I would say something like "Check if object rotation should be in local space instead of world space". By the way, which local space is it in? The local space of the transform target, or the local space of object that this script is on?

I wonder if we want to have an enum "constraint transform space" instead of a bool, something like:

Local
World
Custom / Specified by custom transform.

Then we could use that enum both for rotation, and translation.

Reply:
Yeah, well I went down the path of a custom pivot, and that proved to be quite hard to get the behavior right, so limited the scope of this fix. This is the local space of the transform we are manipulating, will update.